database/src/main/scala/cromwell/database/slick/BackendKVStoreSlickDatabase.scala

```diff
@@ -0,0 +1,24 @@
+package cromwell.database.slick
+
+import cromwell.database.sql.BackendKVStoreSqlDatabase
+import cromwell.database.sql.tables.BackendKVStore
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait BackendKVStoreSlickDatabase extends BackendKVStoreSqlDatabase {
+  this: SlickDatabase =>
+
+  import dataAccess.driver.api._
+
+
+  override def addBackendJobKeyValuePair(entries: Iterable[BackendKVStore])(implicit ec: ExecutionContext): Future[Unit] = {
+    val action = dataAccess.backendKVStoreAutoInc ++= entries
+    runTransaction(action) map { _ => () }
```

Comment:
While I don't know that I'd call it `void` (but I don't know what I *would* call it), I certainly like it better than `map { _ => () }`